sphinx/builders/linkcheck.py

```diff
@@ -26,7 +26,7 @@
 from docutils import nodes
 from docutils.nodes import Element
 from requests import Response
-from requests.exceptions import HTTPError, TooManyRedirects
+from requests.exceptions import HTTPError, TooManyRedirects, ConnectionError
```

Comment:
Can keep alphabetical ordering.
```suggestion
from requests.exceptions import ConnectionError, HTTPError, TooManyRedirects
```